### PR TITLE
Renamed OCS to ODF

### PR DESCRIPTION
**Pull request by Shirisha-S-Rao-1**

### CODE REVIEW

File: config-templates/redhat/odf-remote/4.7/custom-parameters.json [hunk 1]
```diff
@@ -43,22 +43,22 @@
...
+    "description": "Set to 'true' if you want to upgrade the ODF version.",
+    "displayname": "ODF upgrade",
+    "name": "odf-upgrade",
     "default": "false",
     "required": "false",
     "type": "boolean"
   },
   {
-    "description": "The billing type for your OCS configuration.",
+    "description": "The billing type for your ODFconfiguration.",
```
Comment (akgunjal) [inline config-templates/redhat/odf-remote/4.7/custom-parameters.json:61]: Add a space between ODFconfiguration

File: config-templates/redhat/odf-remote/4.7/deployment.yaml [hunk 2]
```diff
@@ -293,7 +293,7 @@ items:
               value: stable-4.7
             - name: WATCH_NAMESPACE
               value: openshift-storage
-            image: "icr.io/obs/ibm-ocs-operator:v4.7.4"
+            image: "icr.io/obs/ibm-ocs-operator:v4.7.5"
```
Comment (akgunjal) [inline config-templates/redhat/odf-remote/4.7/deployment.yaml:296]: Update image to `v4.7.6`

File: config-templates/redhat/odf-remote/4.7/metadata.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "4.7",
   "enabled": "true",
-  "description": "OCS template for openshift 4.7 (Remote storage)",
+  "description": "ODFtemplate for openshift 4.7 (Remote storage)",
```
Comment (akgunjal) [inline config-templates/redhat/odf-remote/4.7/metadata.json:4]: Add a space between ODFtemplate

**Review by akgunjal — CHANGES_REQUESTED** [review]
Check if we need the changes for 4.6 version as well as its still actively supported. Also need image update for 4.6 for PSIRT

File: config-templates/template_list.json [hunk 4]
```diff
@@ -40,17 +40,17 @@
     "provider": "netapp"
   },
   {
-    "name": "ocs-local",
-    "displayname": "OpenShift Container Storage Operator for local devices",
-    "Description": "OpenShift Container Storage Operator for local devices.",
+    "name": "odf-local",
+    "displayname": "OpenShift Data Foundation storage for local devices",
```
Comment (derekpoindexter) [inline config-templates/template_list.json:44]:
```suggestion
    "displayname": "OpenShift Data Foundation for local devices",
```

File: config-templates/template_list.json [hunk 5]
```diff
@@ -40,17 +40,17 @@
...
+    "Description": "OpenShift Data Foundation storage for local devices.",
```
Comment (derekpoindexter) [inline config-templates/template_list.json:45]:
```suggestion
    "Description": "OpenShift Data Foundation for local devices.",
```

File: config-templates/template_list.json [hunk 6]
```diff
@@ -40,17 +40,17 @@
...
     "default-version": "4.7",
     "enabled": "true",
     "provider": "redhat"
   },
   {
-    "name": "ocs-remote",
-    "displayname": "OpenShift Container Storage Operator for remote storage",
-    "Description": "OpenShift Container Storage Operator for remote storage.",
+    "name": "odf-remote",
+    "displayname": "OpenShift Data Foundation storage for remote storage",
```
Comment (derekpoindexter) [inline config-templates/template_list.json:52]:
```suggestion
    "displayname": "OpenShift Data Foundation for remote storage",
```

File: config-templates/template_list.json [hunk 7]
```diff
@@ -40,17 +40,17 @@
...
+    "Description": "OpenShift Data Foundation storage for remote storage.",
```
Comment (derekpoindexter) [inline config-templates/template_list.json:53]:
```suggestion
    "Description": "OpenShift Data Foundation for remote storage.",
```

**Review by akgunjal — APPROVED** [review]
LGTM

**Review by nkkashyap — APPROVED** [review]
Looks good

**Pull request merged**